subprojects/gradle/slack/src/main/kotlin/com/avito/slack/SlackClient.kt

```diff
@@ -5,29 +5,29 @@ package com.avito.slack
 
 import com.avito.android.Result
 import com.avito.slack.model.FoundMessage
-import com.avito.slack.model.SlackChannel
+import com.avito.slack.model.SlackChannelId
 import com.avito.slack.model.SlackMessage
 import com.avito.slack.model.SlackSendMessageRequest
-import com.avito.slack.model.strippedName
 import com.github.seratch.jslack.Slack
 import com.github.seratch.jslack.api.methods.MethodsClient
 import com.github.seratch.jslack.api.methods.SlackApiResponse
-import com.github.seratch.jslack.api.methods.request.channels.ChannelsHistoryRequest
-import com.github.seratch.jslack.api.methods.request.channels.ChannelsListRequest
 import com.github.seratch.jslack.api.methods.request.chat.ChatPostMessageRequest
 import com.github.seratch.jslack.api.methods.request.chat.ChatUpdateRequest
-import com.github.seratch.jslack.api.model.Channel
+import com.github.seratch.jslack.api.methods.request.conversations.ConversationsHistoryRequest
 import java.io.File
 
 interface SlackClient : SlackMessageSender, SlackFileUploader {
 
     fun updateMessage(
-        channel: SlackChannel,
+        channelId: SlackChannelId,
```

Comment:
The idea - get Channel info by id and compare actual and expected Channel name

Reply:
ID should be stable for public channels, names could vary
I don't know if it make sense to compare names and id
i only see value of providing name as a descriptive field (readability of configs);
 but if channel name was changed this description will be outdated; so comment should be enough;

i'll add name for now; let's decide if it was worth - later